apps/api/src/infrastructure/memory/vector_crud_service.py

```
@@ -0,0 +1,89 @@
+            collection = self.client.collections.get(collection_name)
+            collection_with_tenant = collection.with_tenant(user_id)
+
+            collection_with_tenant.data.delete_many(where=Filter.by_property(target).equal(key))
+            logger.info(f"Deleted memory from {collection_name} where {target}={key}")
+
+        except Exception as e:
+            logger.error(f"メモリ削除エラー: {str(e)}")
+            raise
+
+    @retry_on_error(max_retries=2)
+    def update_memory(self, user_id: str, collection_name: str, target: str, key: str, properties: dict, vector: list[float]) -> None:
```

Pylint (3.3.7)

[refactor] Too many arguments (7/5)

(R0913)